### PR TITLE
feat: create result page with data of the api.

### LINKED ISSUE #52: Redirecionar usuário para tela de resultado, carregar os dados de resultado e demonstrar na página

### PR DESCRIPTION
**Pull request by victor-carvalho1996**
# Description

Created the function to get the election results in API notion and show in page results.

## Changes

Screenshot from 2022-12-14 21-05-42

## Board issue

- Closes #52

### CODE REVIEW

File: packages/notion/utils.ts [hunk 1]
```diff
@@ -95,3 +98,55 @@ function _selectHandler(propertyValue: Select): string {
...
+  votes.map((x) => {
+    const result: number = (x * 100) / totalVotes;
+    percentage.push(result.toFixed(2));
+  });
+
+  partiesResult.map(function (value, index) {
+    const currentParty = resultParty.filter((party) => party.name === value);
+
+    results.push({
+      candidate: currentParty[0].members.candidate.name,
```
Comment (vzsoares) [inline packages/notion/utils.ts:143]:
```suggestion
    const [currentParty] = resultParty.filter((party) => party.name === value);

    results.push({
      candidate: currentParty.members.candidate.name,
```
da para simplificar levemente aq

Comment (victor-carvalho1996) [reply packages/notion/utils.ts]: Boa ideia @vzsoares! Melhoria aplicada.

**Review by vzsoares — APPROVED** [review]
lgtm!

**Comment by frattezi** [comment]
Stale

**Pull request closed without merging**